nbs/models.kan.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>Why have only 1 hidden layer? the paper states more layers can be stacked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done! Users can specify an int or a list to hidden_size and specify the number of hidden layers.

If an integer is used, it will assign the same size to all hidden layers.

If a list is used, we check to ensure it has the same number of elements as the number of hidden layers specified.